packages/xrpl/src/models/methods/getAggregatePrice.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+export interface Oracle {
+  account: string
+  oracle_document_id: number
+}
+
+/**
+ * The `get_aggregate_price` method retrieves the aggregate price of specified Oracle objects,
+ * returning three price statistics: mean, median, and trimmed mean.
+ * Returns an {@link GetAggregatePriceResponse}.
+ *
+ * @category Requests
+ */
+export interface GetAggregatePriceRequest extends BaseRequest {
+  command: 'get_aggregate_price'
+
+  base_asset: string
+
+  quote_asset: string
+
+  oracles: Oracle[]
```

Comment:
```suggestion
  oracles: {
    account: string
    oracle_document_id: number
  }[]
```
Having another type called `Oracle` would be confusing.